src/Faker/Provider/id_ID/Company.php

```diff
@@ -16,6 +16,16 @@ class Company extends \Faker\Provider\Company
      */
     protected static $companyPrefix = ['PT', 'CV', 'UD', 'PD', 'Perum'];
 
+    /**
+     * @see https://www.amesbostonhotel.com/macam-macam-profesi-pekerjaan/
+     */
+    protected static $jobTitleFormat = [
```

Comment:
Can we sort the names as mention in the contribution guide?

Reply:
Done sorted the list